docs/source/intro.rst

```diff
@@ -0,0 +1,72 @@
+.. _intro:
+
+Introduction: why is it complex to assess DEM accuracy and precision?
+=====================================================================
+
+Digital Elevation Models are numerical representations of elevation. They are generated from different instruments (e.g., radiometer, radar, lidar), acquired in different conditions (e.g., ground, airborne, satellite), and using different post-processing techniques (e.g., stereophotogrammetry, interferometry).
```

Comment:
Technically, a DEM only one type of numerical representation of elevation. Others are contour lines, meshes and point clouds. Perhaps it is worth adding something along this on the next line (or refer to a more technical description somewhere else):
`Specifically, a DEM is a georeferenced image, where each pixel represents the elevation of that point.`

Reply:
Or I often say a DEM is a **gridded** representation of elevation.

Reply:
Are DEM really generated from radiometers?? That's at least not the first one that comes to my mind! I would just say, "optical sensor" maybe.

Reply:
Also, I would just say "photogrammetry" (without the stereo)

Reply:
I know this is maybe overkill, but you can have  a gridded mesh of elevation too! When people say DEMs, they always mean an array with georeferencing, no?

Anyway, the way you've put it is great @rhugonnet. I'm just doing my best to be annoying!

Reply:
I still think we should add the "gridded" term, as a DEM is always gridded (even though this may apply to other objects too).
Well, actually Wikipedia says:"A DEM can be represented as a raster (a grid of squares, also known as a heightmap when representing elevation) or as a vector-based triangular irregular network (TIN). "

Reply:
It's been added ;)